index.html

```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>New Electron App</title>
```

Comment:
Creo que aqui deberia decir el nombre del proyecto

Reply:
Ok i will implement the comments made